lib/mayaHydra/hydraExtensions/adapters/cameraAdapter.cpp

```diff
@@ -102,8 +102,8 @@ void MayaHydraCameraAdapter::CreateCallbacks()
         dag,
         +[](MObject& transformNode, MDagMessage::MatrixModifiedFlags& modified, void* clientData) {
             auto* adapter = reinterpret_cast<MayaHydraCameraAdapter*>(clientData);
-            adapter->MarkDirty(HdCamera::DirtyTransform);
             adapter->InvalidateTransform();
+            adapter->MarkDirty(HdCamera::DirtyTransform);
```

Comment:
There were a couple places where we were dirtying the transform before invalidating it. I've unified all instances of this sequence of operations to first invalidate the transform and then dirty it.